src/proj_internal.h

```diff
@@ -453,6 +453,27 @@ enum class TMercAlgo {
     PODER_ENGSAGER,
 };
 
+enum class AuxLat {
+  START = 0,
+  GEOGRAPHIC = 0,
+  PARAMETRIC = 1,
+  GEOCENTRIC = 2,
+  RECTIFYING = 3,
+  CONFORMAL  = 4,
+  AUTHALIC   = 5,
+  AUXNUMBER  = 6,
+  PHI = GEOGRAPHIC,
+  BETA = PARAMETRIC,
+  THETA = GEOCENTRIC,
+  MU = RECTIFYING,
+  CHI = CONFORMAL,
+  XI = AUTHALIC,
+  COMMON = GEOGRAPHIC,
+  GEODETIC = GEOGRAPHIC,
+  REDUCED = PARAMETRIC,
+  AUXORDER   = 6,
```

Comment:
Doesn't do a whole lot, but follows the pattern from lines above:
```suggestion
  AUXORDER = AUXNUMBER,
```

Reply:
I've clarified this.  (For the record AUXORDER and AUXNUMBER are distinct quantities only accidently are they equal.)